src/main/java/com/minecolonies/coremod/util/EntityUtils.java

```diff
@@ -40,6 +42,27 @@
      */
     private EntityUtils()
     {
+
+    }
+
+    /**
+     * Checks if a player is a fakePlayer and tries to get the owning player if possible.
+     * @param player the incoming player.
+     * @param world the world.
+     * @return the EntityPlayer owner in the best case.
+     */
+    @NotNull
+    public static EntityPlayer getPlayerOfFakePlayer(@NotNull final EntityPlayer player, @NotNull final World world)
+    {
+        if(player instanceof FakePlayer)
+        {
+            EntityPlayer tempPlayer = world.getPlayerEntityByUUID(player.getUniqueID());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Local variable 'tempPlayer' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)